src/Razor/test/testapps/AppWithPackageAndP2PReference/Views/Home/Contact.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿@{
```

Comment:
Can we have one file - `Index.cshtml`? We don't need all of these to test

Reply:
I took out everything else